TODO

```diff
@@ -16,7 +16,7 @@ commit message:
 [TODO]: Added more tasks for S02.
 
 This way, people working on the test suite can easily spot these commits
-in #perl6.
+in #raku.
```

Comment:
Not totally clear what the hashtag is about. If it's a hashtag, in Twitter #rakulang is used. And apparently in Rosettacode it should be capitalized.

Reply:
I think this is about irc channel, though.